pkg/schedule/coordinator.go

```diff
@@ -312,6 +312,38 @@ func (c *Coordinator) drivePushOperator() {
 	}
 }
 
+// driveEvictSlownNodeScheduler is used to create evciting slow node scheduler if using raft-kv2 engine.
+func (c *Coordinator) driveEvictSlownNodeScheduler() {
```

Comment:
`pd-ctl` allows users to add or remove a scheduler through commands, this way make pd ctl be invalid. 
consider defining it in here: https://github.com/tikv/pd/blob/346e7716e2598dfba2db6afa73c3731e15449f49/pkg/schedule/config/config.go#L522-L531

but there is no logic like `if raftkv2 then defaultV2, else then defaultV1`, we may need add this logic.

Reply:
> but there is no logic like if raftkv2 then defaultV2, else then defaultV1, we may need add this logic.
When initializing schedulers, we cannot know whether the cluster is using raft-kv2 or raft-kv1. Only if the cluster loaded the stores with `raft-kv2` later, can it know whether the cluster is using raft-kv2.